package/Dockerfile

```diff
@@ -17,12 +17,12 @@ RUN zypper -n install autoconf bison curl cmake doxygen make git gcc-c++ flex Me
     tar gzip dbus-1-devel lsb-release graphviz-devel libnsl-devel libcurl-devel libjson-c-devel libacl-devel && \
     rm -rf /var/cache/zypp/*
 
-RUN curl -L https://github.com/rancher/nfs-ganesha/archive/refs/tags/v5_20240430.tar.gz | tar zx \
-	  && curl -L https://github.com/nfs-ganesha/ntirpc/archive/v5.0.tar.gz | tar zx \
-	  && mv nfs-ganesha-5_20240430 nfs-ganesha-5.7 \
-	  && rm -r nfs-ganesha-5.7/src/libntirpc \
-	  && mv ntirpc-5.0 nfs-ganesha-5.7/src/libntirpc
-WORKDIR /nfs-ganesha-5.7
+RUN curl -L https://github.com/rancher/nfs-ganesha/archive/refs/tags/v5_20240716.tar.gz | tar zx \
+	  && curl -L https://github.com/nfs-ganesha/ntirpc/archive/refs/tags/v5.8.tar.gz | tar zx \
```

Comment:
Is `ntiprc` independently released from nfs-ganesha release? so there is no one in the release instead?

Reply:
Yes, `ntiprc` is an independent release. We always need to copy it to nfs-ganesha.
I think ntiprc 5.0 is too old, so I update it together.